Update `Official tournament support` with new badge guidelines (#10902)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

- It must clearly display the logo, motif, or name of your tournament.
- It must be cleanly designed and of reasonable visual quality.
- It must not include any sort of sponsorship or promote anything besides the tournament.

> - It must not include any assets which were not specifically made for the tournament and adheres to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).

Reviewer: "It must not include any assets which [...] adheres to the content usage guidelines"

```suggestion
- Any assets must be specifically made for the tournament and adhere to the [content usage guidelines](/wiki/Rules/Content_usage_permissions).
```

Author: Indifferent to actual wording, original was chosen as it is more consistent with the rest of the article points formatting-wise.

Reviewer 2: avoiding the double-negative is def more important in this case, it can be confusing because "and adheres [...]" could be read as coming *after* the entire "not [...]" clause rather than inside of it

> - It must not include any characters or logos which were not specifically made for the tournament and adheres to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).

Reviewer 3: ```suggestion
- It must not include any media which was not specifically made for the tournament and must adhere to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).
```
thoughts on this wording

Reviewer: "media" isn't really a correct substitution. if you want one word it'd be something like "artworks" or "designs"

(also note the plural)

Reviewer 3: perhaps assets then, since id imagine things like “hype videos” as you call it would be covered too

Author: Please keep in mind that these changes are being specifically made under the "Badges" header so we're not trying to cover every design possible here.

Reviewer 3: Ah my mistake, then assets alone seems like good wording I think to cover many bases

>   - This does not include assets that are open-source or public-use (i.e. fonts, icons, etc).

Reviewer: ```suggestion
  - This does not include assets that are open source or public-use (i.e. fonts, icons, etc).
```

>   - This does not include assets that are open-source or public-use (i.e. fonts, icons, etc).

Reviewer 2: i dont think this should be written as a sub-bullet because it isn't really related. you could also have assets that aren't meant for public use but were made for the tournament for example. also, fonts and icons don't generally fall into this category so idk why they are used to explain this

looking at the next sub-bullet as well, I feel like all of this is kind of unnecessary to write when you've already linked to content usage guidelines, which gets the same points across

Author: content usage guidelines do not cover paid font usage or licensing which these are meant to cover it is very barebones.
